src/features/lireFlow/components/TextReader.tsx

```diff
@@ -39,21 +40,62 @@ const TextReader = () => {
       setError("Erro ao ler o arquivo");
     };
   };
+  const startButton = () => {
+    //Start button logic

```

Comment:
Add this like a todo
```suggestion
    //Todo: Create logic for start button

```